packages/core/lib/test-utils.test.js

```diff
@@ -0,0 +1,66 @@
+import {
+  mockTransaction,
+  resetMockTransaction,
+  waitFor,
+} from '@docknetwork/wallet-sdk-core/lib/test-utils';
+import {ApiRpc} from './client/api-rpc';
+import {getRpcEventEmitter} from './events';
+
+const emitter = getRpcEventEmitter();
+
+describe('Test utils', () => {
+  describe('mock transaction', () => {
+    const hash = '0x1234';
+    const fee = 2;
+    const delay = 100;
+    const txInput = {
+      toAddress: 'test',
+      fromAddress: 'test',
```

Comment:
generally I'd recommend using different test data to make it easier to detect cross-over errors where vars/properties might get mixed up

Reply:
fixed